projects/assets/src/translations/index.ts

```diff
@@ -0,0 +1,13 @@
+import { en } from './en/index';
+
+interface TranslationResources {
```

Comment:
Please consider move it to another file.

Reply:
The lib is so simple that I think that is not required at the moment